calico/felix/plugins/fiptgenerator.py

```diff
@@ -749,6 +875,16 @@ def _build_to_or_from_chain(self, ip_version, endpoint_id, profile_ids,
                 # If the policy accepted the packet, it sets the Accept
                 # MARK==1. Immediately RETURN the packet to signal that it's
                 # been accepted.
+                if not default_drop:
```

Comment:
This wasn't what I expected a flag called `default_drop` to do; I think it'd be worth using a second argument to avoid confusion.